taxi/tests/test_forms.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTests(TestCase):
+    def test_driver_creation_form_is_valid(self):
+        form_data = {
+            "username": "hoakin",
+            "password1": "a0s9d8f7",
+            "password2": "a0s9d8f7",
+            "first_name": "Hoa",
+            "last_name": "Kin",
+            "license_number": "PLO19283"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
The `assertEqual` statement compares `form.cleaned_data` with `form_data`. Note that `cleaned_data` may contain additional fields or transformations (e.g., password hashing), which could cause this assertion to fail. Consider checking only the relevant fields or using `assertDictContainsSubset` if you want to ensure specific fields are present and correct.